### PR TITLE
Task/hw 56153 fx rate changed

### PR DESCRIPTION
**Pull request by ddhar-hw**

### CODE REVIEW

File: Transfer/Sources/ScheduleTransferController.swift [hunk 1]
```diff
@@ -88,6 +98,39 @@ extension ScheduleTransferController {
...
+                return nil
+        }
+        view.footerLabel.attributedText = attributedText
+        return view
+    }
+
+    private func getAttributedFooterText(for section: Int) -> NSAttributedString? {
+        let sectionData = presenter.sectionData[section]
+        var attributedText: NSAttributedString?
+        if  let transferSectionData = sectionData as? ScheduleTransferSummaryData {
```
Comment (jsaini-hw) [inline Transfer/Sources/ScheduleTransferController.swift]: double space between ``` if  let ```

File: Transfer/Sources/ScheduleTransferController.swift [hunk 2]
```diff
@@ -109,6 +152,7 @@ extension ScheduleTransferController {
             if let tableViewCell = cell as? TransferSummaryCell,
                 let summaryData = section as? ScheduleTransferSummaryData {
                 tableViewCell.configure(summaryData.rows[indexPath.row].title, summaryData.rows[indexPath.row].value)
+                addFooter()
```
Comment (jsaini-hw) [inline Transfer/Sources/ScheduleTransferController.swift]: why do you need this?

File: Transfer/Sources/ScheduleTransferController.swift [hunk 3]
```diff
@@ -130,6 +174,14 @@ extension ScheduleTransferController {
     private func tapScheduleTransfer(sender: UITapGestureRecognizer) {
         presenter.scheduleTransfer()
     }
+
+    func addFooter() {
```
Comment (jsaini-hw) [inline Transfer/Sources/ScheduleTransferController.swift]: No need of this

Comment (ddhar-hw) [reply Transfer/Sources/ScheduleTransferController.swift]: You mean put this code in the switch statement itself - inside `getCellConfiguration`

Comment (jsaini-hw) [reply Transfer/Sources/ScheduleTransferController.swift]: No we do not need it at all. ``` override public func tableView(_ tableView: UITableView, viewForFooterInSection section: Int) -> UIView?  ``` will add the footer. Also fix all the swiftlint warning too

**Pull request merged**